compatibility/core/src/main/java/org/mule/compatibility/core/transport/AbstractConnector.java

```diff
@@ -405,17 +405,18 @@ public final synchronized void start() throws MuleException {
       return;
     }
 
-    scheduler = createScheduler();
-
     if (!isConnected()) {
+      scheduler = createScheduler();
```

Comment:
why should a scheduler only be created if not connected, and how would a connector already be connected on the start() method?